pose_evaluation/metrics/conftest.py

```diff
@@ -0,0 +1,34 @@
+import shutil
```

Comment:
what is this file?

Reply:
It's a Pytest thing for creating fixtures that are reusable across different files, I just learned about it recently. https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
but currently there is a single test file?
previously, i used [unittest](https://github.com/sign-language-processing/signwriting-evaluation/blob/main/signwriting_evaluation/metrics/test_bleu.py#L6) which i found easier to work with, but these fixtures are good too.

Reply:
Yeah, pytest automatically makes things in conftest available in other files, automagically. So as we add metrics and tests and want to, say, check the range, we can use this